spring-ai-docs/src/main/antora/modules/ROOT/nav.adoc

```diff
@@ -56,49 +57,48 @@
 **** xref:api/embeddings/vertexai-embeddings-palm2.adoc[PaLM2 Embedding]
 *** xref:api/embeddings/watsonx-ai-embeddings.adoc[watsonx.AI]
 *** xref:api/embeddings/zhipuai-embeddings.adoc[ZhiPu AI]
-** xref:api/imageclient.adoc[]
+** xref:api/imageclient.adoc[Image Models]
 *** xref:api/image/azure-openai-image.adoc[Azure OpenAI]
 *** xref:api/image/openai-image.adoc[OpenAI]
 *** xref:api/image/stabilityai-image.adoc[Stability]
 *** xref:api/image/zhipuai-image.adoc[ZhiPuAI]
 *** xref:api/image/qianfan-image.adoc[QianFan]
-** xref:api/audio[Audio Model API]
+** xref:api/audio[Audio Models]
 *** xref:api/audio/transcriptions.adoc[]
 **** xref:api/audio/transcriptions/azure-openai-transcriptions.adoc[Azure OpenAI]
 **** xref:api/audio/transcriptions/openai-transcriptions.adoc[OpenAI]
 *** xref:api/audio/speech.adoc[]
 **** xref:api/audio/speech/openai-speech.adoc[OpenAI]
-** xref:api/moderation[Moderation Model API]
+** xref:api/moderation[Moderation Models]
 *** xref:api/moderation/openai-moderation.adoc[OpenAI]
+// ** xref:api/generic-model.adoc[]
 
-** xref:api/vectordbs.adoc[]
-*** xref:api/vectordbs/azure.adoc[]
-*** xref:api/vectordbs/apache-cassandra.adoc[]
-*** xref:api/vectordbs/chroma.adoc[]
-*** xref:api/vectordbs/elasticsearch.adoc[]
-*** xref:api/vectordbs/gemfire.adoc[GemFire]
-*** xref:api/vectordbs/milvus.adoc[]
-*** xref:api/vectordbs/mongodb.adoc[]
-*** xref:api/vectordbs/neo4j.adoc[]
-*** xref:api/vectordbs/opensearch.adoc[]
-*** xref:api/vectordbs/oracle.adoc[Oracle]
-*** xref:api/vectordbs/pgvector.adoc[]
-*** xref:api/vectordbs/pinecone.adoc[]
-*** xref:api/vectordbs/qdrant.adoc[]
-*** xref:api/vectordbs/redis.adoc[]
-*** xref:api/vectordbs/hana.adoc[SAP Hana]
-*** xref:api/vectordbs/typesense.adoc[]
-*** xref:api/vectordbs/weaviate.adoc[]
-
-** xref:api/functions.adoc[Function Calling]
-** xref:api/multimodality.adoc[Multimodality]
-** xref:api/prompt.adoc[]
-** xref:api/structured-output-converter.adoc[Structured Output]
+* xref:api/vectordbs.adoc[]
 ** xref:api/etl-pipeline.adoc[]
-** xref:api/testing.adoc[]
-** xref:api/generic-model.adoc[]
+** xref:api/vectordbs/azure.adoc[]
```

Comment:
Similar to the other comment, the ETL pipeline, while certainly being vector store focused, is it's own top level concern or concept and like the other AI Model sections, this is where I go to find specific info on a vendor's vector store as compared to the higher level topic of ETL.